### PR TITLE
[User] Handle delete product cart

### PR DESCRIPTION
**Pull request by truongha-1240**
## Related Tickets
- [#47508](https://edu-redmine.sun-asterisk.vn/issues/47508)

## WHAT (optional)
- Delete a product in cart

## Evidence (Screenshot or Video)
image
image
image

### CODE REVIEW

File: app/views/shared/_item_cart.html.erb [hunk 1]
```diff
@@ -22,6 +22,6 @@
   <td><%= number_to_currency(product_detail.price) %></td>
   <td class="font-weight-bold"> <%= number_to_currency(product_detail.price*quantity) %> </td>
   <td>
-    <%= link_to t("delete"), "#", method: :delete, class: "btn btn-danger" %>
+    <%= link_to content_tag( :i, nil, :class=>"fa fa-trash fa-lg"), cart_path(product_detail.id), remote: true, method: :delete, class: "btn btn-danger", :data => {:confirm => 'Are you sure?'} %>
```
Comment (huongnt-2545) [inline app/views/shared/_item_cart.html.erb]: - dòng dài rứa em hè, xuống dòng cho hợp lý nhé
- `:class=>"fa fa-trash fa-lg"` => `class: "fa fa-trash fa-lg"`
-  I18n

Comment (truongha-1240) [reply app/views/shared/_item_cart.html.erb]: update ạ

**Comment by truongha-1240** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**